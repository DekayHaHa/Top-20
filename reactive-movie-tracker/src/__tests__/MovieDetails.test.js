import React from 'react';
import { shallow } from "enzyme";
import { MovieDetails }
from '../components/MovieDetails'


describe("MovieDetails", () => {
	let wrapper;
	beforeEach(() => {
		wrapper = shallow(<MovieDetails />);
	});


	it("should match the snapshot with all data passed in correctly", () => {
		expect(wrapper).toMatchSnapshot();
	});

});