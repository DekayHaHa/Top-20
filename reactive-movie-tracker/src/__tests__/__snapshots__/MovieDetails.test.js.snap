// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails should match the snapshot with all data passed in correctly 1`] = `
<div
  className="movie-details"
>
  <img
    alt="movie poster"
    className="img-details"
  />
  <div
    className="details-container"
  >
    <h1
      className="title-details"
    />
    <p
      className="score-details"
    >
      Score: 
    </p>
    <p
      className="overview-details"
    />
  </div>
</div>
`;
